src/main/kotlin/agallipo/main.kt

```diff
@@ -1,12 +1,12 @@
-package agallipo
+    package agallipo
 
+    //created by alex :)
+    fun main(args: Array<String>) {
+        val pathname = "examples/example.json"
+        var boardingCardJsonParser = BoardingCardJsonParser(pathname)
+        var flightPaths = boardingCardJsonParser.sortTickets(boardingCardJsonParser.boardingCards)
```

Comment:
I think these variables "boardingCardJsonParser" and "flightPaths" do not change, right? In that case they can be declared with "val" instead of "var"

Reply:
Yep thanks! I took your advice and all of my main's variables are val